### ISSUE TITLE
byte_size does not work on binaries that started via "" elixir string syntax

### ISSUE THREAD

**Issue by vans163** [elixirscript/elixirscript#384]
```
#Good
Console.log byte_size(<<1,2,3>>)
3

#Bad
Console.log byte_size("\x01\x02\x03")
undefined

#Bad
Console.log byte_size("abc")
undefined
```

**Comment by bryanjos** [comment]
I think this can be resolved by checking if a string is given and returning the length, otherwise do what is being done now

**Comment by vans163** [comment]
https://github.com/elixirscript/elixirscript/pull/387/commits/b6bb83cb2bca3f0c82391b7dfccee3df072d8e7f fixed.  But I do not like all these little hotpatch fixes, I think there maybe a lot more of them.

**Comment by bryanjos** [comment]
closed with #387
